pygmt/src/grdlandmask.py

```diff
@@ -45,6 +49,38 @@ def grdlandmask(**kwargs):
         in.
     {I}
     {R}
+    {A}
+    resolution : str
+        *res*\[\ **+f**\]. Selects the resolution of the data set to use
+        ((**f**)ull, (**h**)igh, (**i**)ntermediate, (**l**)ow, or
+        (**c**)rude). The resolution drops off by ~80% between data sets.
+        [Default is **l**]. Append **+f** to automatically select a lower
+        resolution should the one requested not be available
+        [abort if not found]. Alternatively, choose (**a**)uto to automatically
+        select the best resolution given the chosen region. Note that because
+        the coastlines differ in details a node in a mask file using one
+        resolution is not guaranteed to remain inside [or outside] when a
+        different resolution is selected.
+    bordervalues : bool or str or list
+        Nodes that fall exactly on a polygon boundary should be
+        considered to be outside the polygon [Default considers them to be
+        inside]. Alternatively, append either the four values *cborder*/\
+        *lborder*/*iborder*/*pborder* or just the single value *bordervalue*
+        (for the case when they should all be the same value). This turns on
+        the line-tracking mode. Now, after setting the mask values specified
+        via ``maskvalues`` we trace the lines and change the node values for
+        all cells traversed by a line to the corresponding border value. Here,
+        *cborder* is used for cells traversed by the coastline, *lborder* for
+        cells traversed by a lake outline, *iborder* for islands-in-lakes
+        outlines, and *pborder* for ponds-in-islands-in-lakes outlines
+        [Default is no line tracing].
+    maskvalues : str or list
+        *values*. Sets the values that will be assigned to nodes. Values can
+        be any number, including the textstring NaN. Also select
+        ``bordervalues`` to let nodes exactly on feature boundaries be
+        considered outside [Default is inside]. Specify this information
+        using either *wet*/*dry* or *ocean*/*land*/*lake*/*island*/*pond*.
+        [Default is 0/1/0/1/0 (i.e., 0/1)].
```

Comment:
Perhaps write the docstring in this way is better.
```suggestion
        [*wet*, *dry*] or [*ocean*, *land*, *lake*, *island*, *pond*]. 
        Sets the values that will be assigned to nodes. Values can
        be any number, including the textstring NaN 
        [Default is 0/1/0/1/0 (i.e., 0/1)]. Also select
        ``bordervalues`` to let nodes exactly on feature boundaries be
        considered outside [Default is inside]. 
```